personal.rb

```diff
@@ -0,0 +1,23 @@
+"ruby.rubocop.executePath": "/home/marta/.rbenv/shims/",
```

Comment:
let's remove it from this PR, it's personal :)

Reply:
@mtrojanowska this is the only change blocking this PR from being merged

Reply:
fixed